packages/edit-site/src/components/sidebar-navigation-screen-template/home-template-details.js

```diff
@@ -1,144 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
```

Comment:
Can you keep this around for a bit? We'll have to move them to inspector controls and it might be helpful to have the existing code. I'll see to open a PR for the next days.

Reply:
If this should block the PR, it's fine we can wait but if it shouldn't, we can always access the code in the old branches.

Reply:
Of course it shouldn't block this PR. I'll try to open a PR for that asap though.